tests/e2e_tests/run

```diff
@@ -31,41 +37,77 @@ parse_args(){
 
     # FIXME args must be passed in the form -f=<path>; spaces don't workalthough they're standard...
     while [ $# -gt 0 ]; do
-        PARAM=`echo $1 | awk -F= '{print $1}'`
-        VALUE=`echo $1 | awk -F= '{print $2}'`
-        case $PARAM in
+        case $1 in
             -h | --help)
             usage
             exit
             ;;
             -c | --skip-cleanup)
             SKIP_CLEANUP=1
+            shift
+            ;;
+            --enterprise)
+            COMPOSE_CMD+=$ENTERPRISE_COMPOSE
+            ENTERPRISE=1
+            shift
             ;;
             -f)
-            COMPOSE_CMD+=( $VALUE )
+            COMPOSE_CMD+=$2
+            shift 2
             ;;
             *)
         esac
-        shift 1
     done
 }
 
+USER=mender-demo@example.com
+PASSWORD=mysecretpassword!123
+
 run_tests() {
+    cat ${INTEGRATION_PATH}/certs/api-gateway/cert.crt ${INTEGRATION_PATH}/certs/storage-proxy/cert.crt > ${GUI_REPOSITORY}/tests/e2e_tests/dockerClient/cert.crt
+    COMPOSE_CMD+=$TEST_COMPOSE
+    # $COMPOSE_CMD down
```

Comment:
Debug?

Reply:
Argh, yes that was for debugging indeed - thanks for catching!